docs/sphinx/using/basics/run_kernel.rst

```diff
@@ -82,6 +82,87 @@ Sample
 
   See the :doc:`API specification <../../../api/languages/cpp_api>` for further information.
 
+Sample Async
```

Comment:
Please add the following paragraph (from [Executing Quantum Circuits](https://nvidia.github.io/cuda-quantum/latest/examples/python/tutorials/executing_circuits.html)) to the top of the page, just before the 'Sample' subtitle: (make sure you keep the same format)
In CUDA-Q, quantum circuits are stored as quantum kernels. For estimating the probability distribution of a measured quantum state in a circuit, we use the sample function call, and for computing the expectation value of a quantum state with a given observable, we use the observe function call.

I think this is a good paragraph to add as you merge the two files into one.